src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -1328,6 +1330,13 @@ public long getRuinedTime() {
 		return ruinedTime;
 	}
 
+	public String getMapColorHexCode() {
+		String rawMapColorHexCode = hasNation() ? nation.getMapColorHexCode() : null;
```

Comment:
Seems like this method can potentially return `null`. The method should probably be annotated with a `@Nullable` then.

Reply:
Makes sense, done.